script/jenkins/test-install-packages.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -xeuo pipefail
 
-mage -v -debug testPackagesInstall
+./build/linux/mage -v testPackagesInstall
```

Comment:
we can use directly `mage` if we merge https://github.com/elastic/apm-server/pull/3456/files#r392167309